src/app/folder/folderlist.component.ts

```diff
@@ -43,6 +44,8 @@ class FolderNode {
 export class FolderListComponent {
     selectedFolder = 'Inbox';
     dropFolderId: number;
+    dropAboveOrBelowOrInside = 0;
```

Comment:
Wouldn't this be better off as an [enum](https://www.typescriptlang.org/docs/handbook/enums.html)?

Reply:
changed to using `enum` now.